.gitignore

```diff
@@ -8,3 +8,5 @@
 /dist
 
 npm-debug.log*
+
+.idea
```

Comment:
It's generally better to add IDE-specific files to a global `.gitignore` (usually at in hour home directory) and then tell git to look at that file with `git config --global core.excludesFile '~/.gitignore'` :)

Reply:
I didn't know about this, thanks for the tip!